components/card/article/src/index.scss

```diff
@@ -13,11 +13,17 @@ $c-article-tag-types: () !default;
   }
 
   &-info {
-    align-items: center;
-    display: flex;
-    justify-content: space-between;
-    margin: 0 $m-h;
-    padding: $p-v 0;
+    .is-featured & {
```

Comment:
Not sure if this is a good solution, because we're defining a global selector, what will happen if there's something like this?:
```html
<div class="app is-featured">
  <!-- Bla, bla, bla... -->
  <div class="sui-CardArticle-infoInner">Text</div>
</div>
```

We should just apply this selector to the specific element.